docs/StyleOverridesGuide.md

```diff
@@ -0,0 +1,227 @@
+## Style Overrides Guide
+
+Material UI, and by extension, Brightlayer UI provides several different ways to customize a component's styles. Your specific context will determine which one is ideal. From narrowest to broadest use case, here are the options:
+
+## 1. The `sx` prop
+
+The [sx prop](https://mui.com/system/basics/#the-sx-prop) is the best option for adding style overrides to a single instance of a component in most cases.
+
+#### Usage
+
+```tsx
+import { InfoListItem } from '@brightlayer-ui/react-components';
+...
+
+<InfoListItem
+    title={'Info List Item'}
+    sx={{ my: 2, backgroundColor: '#efefef' }}
+/>
+
+```
+
+### Overriding nested component styles
+
+To customize a specific part of a component, you can use the global CSS class provided by Brightlayer UI inside the sx prop\*. These global CSS class names can be found in the `Style Overrides` section of the documentation for each component. As an example, let's say you want to customize the InfoListItem component's icon size and padding.
+
+> \* not all classes can be targeted via nesting. Pay attention to the actual DOM structure as some elements live as siblings to the root element and some are in separate containers entirely.
+
+#### Usage
+
+```tsx
+import { InfoListItem } from '@brightlayer-ui/react-components';
+import { Leaf } from '@brightlayer-ui/icons-mui';
+...
+<InfoListItem
+    title={'Info List Item'}
+    icon={<Leaf />}
+    sx={{ width: '100%', '& .BluiInfoListItem-icon' : { ml: '2px', fontSize: '1rem' } }}
+/>
+```
+
+## 2. Styled component
+
+To reuse the same overrides in different locations across your application, create a reusable component using the [styled()](https://mui.com/system/styled/) utility.
+
+#### Usage
+
+```tsx
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import { InfoListItem, InfoListItemProps } from '@brightlayer-ui/react-components';
+import { CheckCircle } from '@mui/icons-material';
+import * as Colors from '@brightlayer-ui/colors';
+...
+const SuccessInfoListItem = styled(InfoListItem)<InfoListItemProps>(({ theme }) => ({
+    color: theme.palette.success.main,
+    '& .BluiInfoListItem-icon': {
+        fontSize: '1.5rem',
+    },
+}));
+
+// usage
+<SuccessInfoListItem
+    title={'Success'}
+    icon={<CheckCircle />}
+    statusColor={Colors.green[500]}
+    iconColor={Colors.green[500]}
+/>;
+
+```
+
+## 3. Overriding Styles with the `className` and `classes` prop
+
+If you want to override a component's styles using custom classes, you can use the `className` prop, available on each component.
```

Comment:
This should say something about the `className` only being applied to the root element of the component.